compiler/src/iree/compiler/Codegen/Utils/Utils.cpp

```diff
@@ -183,6 +187,20 @@ bool hasUkernel(IREE::HAL::ExecutableTargetAttr targetAttr,
   return false;
 }
 
+// TODO: Add more popular kernels into this list and the ukernel cmake.
+//       No real technical reason to only allow these aside from compile
+//       time and diskspace.
+bool hasUkernelSupportedRocmArch(IREE::HAL::ExecutableTargetAttr targetAttr) {
+  auto targetArch = getConfigStringAttr(targetAttr, "target_arch");
+  if (!targetArch) {
+    return false;
+  }
+  StringRef targetArchStr = targetArch->getValue();
+  const llvm::StringSet<> kSupportedTargetChip{"gfx90a", "gfx940", "gfx1030",
```

Comment:
We'd want to use POD in global variables if possible. Here I think just use an array of `const char *` and compare one by one would be fine..

Reply:
But this is not really a global variable right? Maybe I am misunderstanding the term global variable haha.

Reply:
Done, updated to use POD. :)

Reply:
Sorry I meant that variables having a lifetime as the program. In that sense it's just like global variables.

Reply:
Makes sense, thanks :)